x-pack/plugins/rule_registry/server/search_strategy/search_strategy.test.ts

```diff
@@ -199,4 +201,66 @@ describe('ruleRegistrySearchStrategyProvider()', () => {
       .toPromise();
     expect(result).toBe(EMPTY_RESPONSE);
   });
+
+  it('should support pagination', async () => {
+    const request: RuleRegistrySearchRequest = {
+      featureIds: [AlertConsumers.LOGS],
+      pagination: {
+        pageSize: 10,
+        pageIndex: 1,
```

Comment:
Page indexes start at 0, right? Is starting at 1 here intentional? To get the second page of results?

Reply:
Sure!